src/Orleans/Runtime/GrainReference.cs

```diff
@@ -272,6 +272,17 @@ protected virtual int InterfaceId
         /// <summary>
         /// Implemented in generated code.
         /// </summary>
+        protected virtual ushort InterfaceVersion
+        {
+            get
+            {
+                throw new InvalidOperationException("Should be overridden by subclass");
+            }
```

Comment:
This property is only used when `InterfaceId` is used (and this property also throws `InvalidOperationException("Should be overridden by subclass")` by default), so no, it will not conflict